src/aap_eda/settings/default.py

```diff
@@ -391,8 +393,15 @@ def _rq_common_parameters():
             "HOST": REDIS_HOST,
             "PORT": REDIS_PORT,
         }
-        if REDIS_CLIENT_CERT_PATH:
+        if REDIS_TLS:
```

Comment:
I know I didn't mention this yesterday, and I'm not asking for it to be changed, but this could all be collapsed to:
```suggestion
        params["SSL"] = REDIS_TLS or (REDIS_CLIENT_CERT_PATH and REDIS_TLS is None)
```

Reply:
Right!  My goal was to keep the logic "exploded" to make it super obvious what needs to be done.